# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
# Commit: 8d60ee1f7cd7
# Full Hash: 8d60ee1f7cd7cc655c34d66e070feea356634997
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Remove CODEC_PLNAME_SIZE as CodecInst no longer exists upstream. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102277
# ==============================================================================

diff -r f16cf2573a4d -r 8d60ee1f7cd7 dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Mon Jan 18 11:16:22 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Fri Jan 29 17:03:42 2021 +0100
@@ -42,9 +42,6 @@
 #endif
 #define LOGTAG acLogTag
 
-// 32 bytes is what WebRTC CodecInst expects
-const unsigned int WebrtcAudioConduit::CODEC_PLNAME_SIZE = 32;
-
 using LocalDirection = MediaSessionConduitLocalDirection;
 /**
  * Factory Method for AudioConduit
@@ -1035,9 +1032,8 @@
     return kMediaConduitMalformedArgument;
   }
 
-  if ((codecInfo->mName.empty()) ||
-      (codecInfo->mName.length() >= CODEC_PLNAME_SIZE)) {
-    CSFLogError(LOGTAG, "%s Invalid Payload Name Length ", __FUNCTION__);
+  if (codecInfo->mName.empty()) {
+    CSFLogError(LOGTAG, "%s Empty Payload Name ", __FUNCTION__);
     return kMediaConduitMalformedArgument;
   }
 